index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.2/font/bootstrap-icons.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous"/>
+    <link rel="stylesheet" href="style.css">
+    <title>Yunus Ali</title>
+</head>
+<body>
+    <header>
+        <nav>
+            <h4>Yunus Ali</h4>
+            <i class="bi bi-list"></i>
+        </nav>
+    </header>
+    <main>
+        <div>
+        <article>
+            <h2>I’m Yunus
```

Comment:
I also suggest that the introduction content in the` <h2> `tag is put in an `<h1>`. This is important because the h1 is the star of our design - it captures the most attention so it should be emphasized. It can also be useful in case of search engine optimization.